test/game/game_setup_test.exs

```diff
@@ -96,5 +96,18 @@ defmodule GameSetupTest do
       assert players ==
         %Players{player_one: %Player{type: HardComputerPlayer, symbol: "X"}, player_two: %Player{type: HardComputerPlayer, symbol: "O"}}
     end
+
+    test "displays invalid setup validation message via invalid input for player selection, then returns players containing selected players and associated symbols" do
+      user_inputs = ["3", "1", "1"]
```

Comment:
What do these inputs mean? More explicitly how is `3` different from `1`

Reply:
3 would be out of bounds, but them being vague is a really good point! I think I might specify which is the valid and which is the invalid. Something like:

`invalid_input = "3"`
`valid_inputs = ["1", "1"]`

and then either put them together there in the test or have a test helper function that can format the valid and invalid inputs, I think this would improve readability and understanding, what are your thoughts? 😄

Reply:
Yea I recall this is something I had mentioned before. I think we can go a step further than valid or invalid by providing more context. For example:
```ex
user_inputs = [
  invalid_user_selection,
  valid_user_selection,
  selection_easy_computer_mode,
  ...
]
```

Reply:
I like that alot! The only question I have is, is it ok to have all of the inputs assigned before this user_inputs assignment for the sake of readability, or does that make the tests too messy? Or maybe is there a neat/organized way to assign all these before the user_inputs assignment? Im gonna look into this, Thank you 😄

Reply:
I think Im gonna make this its own refactor PR as I want to make test helpers to help along with this, but I think it would make this PR too messy, what are your thoughts?

Reply:
So create the test helper, but just use it in this test for now? Sounds like a plan to me if I understood that correctly!